data/news/platform/ocaml-5.0.rc1.md

```diff
@@ -0,0 +1,79 @@
+---
+title: OCaml 5.0.0 - First release candidate
+description: First release candidate of OCaml 5.0.0
+date: "2022-12-06"
+tags: [ocaml, platform]
+---
+
+
+The release of OCaml version 5.0.0 is imminent. As a final step before the
+release, we are publishing a release candidate that you can test while waiting
+for the release in the upcoming weeks.
+
+If you find any bugs, please report them on [OCaml's issue tracker](https://github.com/ocaml/ocaml/issues).
+
+  https://github.com/ocaml/ocaml/issues
+
+Compared to the second beta release, this releases contains one toplevel bug fix
+and a minor type system fix.
+
+If you are interested by the changes beyond the new multicore runtimes, the
```

Comment:
```suggestion
If you are interested by the changes beyond the new Multicore runtimes, the
```
I’ve been told when referring specifically to OCaml Multicore to capitalised it for emphasis.